test/express.ts

```diff
@@ -117,3 +122,44 @@ Can include **Markdown**.`,
     help: "This is a simple method.\nIt just returns success.",
   });
 });
+
+test("well known services", async (t) => {
+  const app = express();
+
+  const registry = new Registry();
+
+  const service = {
+    hello: (x: { msg: string }) => {
+      console.log("method", x.msg);
+      return `success ${x.msg}`;
+    },
+  };
+  const meta: ServiceDetails<typeof service> = {
+    expose: [{ methodName: "hello" }],
+    service: "hello-service",
+    help: "hello",
+  };
+
+  registry.register(service, meta);
+
+  app.use(
+    "/.well-known/loke-rpc",
```

Comment:
Some "light" reading for standards on this

https://datatracker.ietf.org/doc/html/rfc5785

Reply:
https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml

Reply:
We could have a tiered approach, eg this comment

> The suffix "posh" is expected to be followed by an additional path component consisting of a service name (say, "spice") and a file extension of ".json", resulting in a full path of, for instance, "/.well-known/posh/spice.json". Registration of service names shall be requested by developers of the relevant application protocols.

eg `/.well-known/loke/rpc-server` `/.well-known/loke/rpc-client`

Reply:
Think `/.well-known/loke-rpc/server`  `/.well-known/loke-rpc/client` is good